javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+var $modal = document.querySelector('.modal');
+var $button = document.querySelector('.open-modal');
+var $modalBtn = document.querySelector('.modal-button');
+
+$button.addEventListener('click', function () {
+  $modal.style.display = 'block';
```

Comment:
Oh noooo, this is inline styling, exactly the same as using the `style` attribute on an element 💀 

Instead, I would make a class that you can add or remove to hide/show the element 🤓